lib/packages/docutils/rst.nim

```diff
@@ -2593,14 +2616,14 @@ proc getColumns(p: RstParser, cols: var RstCols, startIdx: int): int =
     if p.tok[result].kind != tkAdornment: break
   if p.tok[result].kind == tkIndent: inc result
 
-proc checkColumns(p: RstParser, cols: RstCols) =
+proc checkColumns(p: RstParser, cols: RstCols): bool =
```

Comment:
What's the point of always returning `true`?

Reply:
silly me. Wanted to abort parsing table in some cases, but then decided it's not worth it. Reverted.